app/splits/[id]/early-game-planning-panel.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Heading, VStack, Text, Divider, Flex, Spacer } from "@chakra-ui/react";
+import ThreeStateSelection from "@/app/components/three-state-selection";
+
+function EarlyGamePlanningPanel({ earlyGamePlanning, onUpdate }: { earlyGamePlanning: EarlyGamePlanning, onUpdate: (earlyGamePlanning: EarlyGamePlanning) => void }) {
+  function onWinLevel1Update(state: boolean | null) {
+    onUpdate({
+      ...earlyGamePlanning,
+      winlvl1: state
+    });
+  }
```

Comment:
Try to simplify this functions to something like
```suggestion
    onUpdate({
      ...earlyGamePlanning,
      ['winlvl2']: state
    });
  
```